libraries/libbls/libbls.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"encoding/hex"
+	"log"
+
+	"github.com/ElrondNetwork/elrond-go-crypto/signing"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing/mcl"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing/mcl/singlesig"
+)
+
+var (
+	keyGenerator = signing.NewKeyGenerator(mcl.NewSuiteBLS12())
+	blsSigner    = singlesig.BlsSingleSigner{}
+)
+
+func doGeneratePrivateKey() string {
```

Comment:
why did we have the "do" prefix? A method is considered an action, anyway.
Also, add a Hex suffix on the method name? `generatePrivateKeyAsHex` ?
Also on L28 -> `generatePublicKeyAsHex` ?

Reply:
Renamed (`do` kept, as discussed).